src/org/zaproxy/zap/spider/URLCanonicalizer.java

```diff
@@ -469,6 +468,11 @@ private static String normalizePath(final String path) {
 		return path.replace("%7E", "~").replace(" ", "%20");
 	}
 
+	private static boolean isSessionToken(String paramName) {
+		return Control.getSingleton().getExtensionLoader().getExtension(ExtensionHttpSessions.class)
```

Comment:
- As mentioned in the issue the extension is null if disabled.
 - This class should not depend on the extension, it should be up to the callers to include this additional check :)

Reply:
So, do I need remove checking for sessionToken totally including hardcoded values which I've already deleted?

Reply:
Yeah, "older" methods should use the old values (it can be used a default predicate with those though).